scripts/dev/build_release.py

```diff
@@ -282,18 +278,18 @@ def build_mac(
         shutil.rmtree(d, ignore_errors=True)
 
     utils.print_title("Updating 3rdparty content")
-    update_3rdparty.run(ace=False, pdfjs=True, legacy_pdfjs=qt5, fancy_dmg=False,
+    update_3rdparty.run(ace=False, pdfjs=True, legacy_pdfjs=False, fancy_dmg=False,
```

Comment:
Unrelated, but note to self: With #8199 and PDF.js potentially continuing to somewhat aggressively use new browser features, should we always ship the legacy build? Or do we have tests to catch this being an issue?

Reply:
If you think it's going to make our lives easier, I vote do it! We can pull in the new one if people want new features.
Hmm, I though pdfjs-legacy was an old version, but the docs say it's just the latest version with polyfills? That's cool! https://github.com/mozilla/pdf.js/wiki/Frequently-Asked-Questions#faq-support

Reply:
Yeah, exactly - it's the same code, just built for Chromium 103+ support according to that FAQ (so QtWebEngine 6.4+ now), rather than only supporting the latest Chromium release. I wasn't actually aware of the "only latest" thing, I thought they at least had some reasonable span of versions they support. Given that, we definitely need the legacy build always!

Reply:
Is this what you envisioned: https://github.com/qutebrowser/qutebrowser/issues/8332

Not sure if I took that a bit far, or if you just were thinking of changing it on this branch or what.

Reply:
No, that is pretty much what I had in mind! :+1: